examples/deployments/oss.py

```diff
@@ -0,0 +1,24 @@
+"""
```

Comment:
Can we name this `mlflow.py` instead of `oss.py`?

Reply:
sg